// Copyright 2022 HDL
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";
package AGE.ProtoBuf.Client;

option cc_generic_services = true;

message SendResultRequest {
    required bytes result = 1;
    optional bool is_throughput_test = 2 [default = false];
};

message SendResultResponse {
    required bool success = 1;
};

service Service {
    rpc SendResult(SendResultRequest) returns (SendResultResponse);
};
